package.json

```diff
@@ -17,6 +17,7 @@
     "react-router-dom": "^6.15.0",
     "react-scripts": "5.0.1",
     "sass": "^1.64.2",
+    "styled-components": "^6.0.8",
```

Comment:
Jak nie korzystasz z styled components to usun

Reply:
done